src/github-web-worker.js

```diff
@@ -167,6 +167,11 @@ const api = {
         content: JSON.stringify(value)
       };
 
+      if (currentFiles && currentFiles.hasOwnProperty(this.oldBackupFileName)) {
+        // found old backup file, it will be deleted
+        files[this.oldBackupFileName] = null;

```

Comment:
The gist API is fast but not blazing fast, if you are testing this functionality you have to give it sometimes up to one minute to start returning the correct json response from the API (the changes you make to the gist are sometimes not available right away via the api). I have tested every flow and it works.